plugins/SegmentEditor/SegmentSelectorControl.php

```diff
@@ -54,8 +54,7 @@ public function __construct($idSite = false)
             if ($segment['category'] == $visitTitle
                 && ($segment['type'] == 'metric' && $segment['segment'] != 'visitIp')
             ) {
-                $metricsLabel = Piwik::translate('General_Metrics');
-                $metricsLabel[0] = Common::mb_strtolower($metricsLabel[0]);
+                $metricsLabel = Common::mb_strtolower(Piwik::translate('General_Metrics'));
```

Comment:
Yes, this is the fix actually: it does not work to use [0] to replace first character  in a multibyte chinese string since the first character might be 2 or 3 bytes.